fineract-doc/src/docs/en/chapters/architecture/fineract-database-handling.adoc

```diff
@@ -0,0 +1,114 @@
+= Fineract Database Handling
+
+== Fineract Database Upgrades
```

Comment:
What's this section title, upgrades? Upgrades of what? Below the text is about the different database supports.